doc/admin-guide/files/records.config.en.rst

```diff
@@ -273,6 +273,31 @@ System Variables
    order of auto-deletion (if enabled). A default value of 0 means auto-deletion will try to keep
    output logs as much as possible. See :doc:`../logging/rotation.en` for guidance.
 
+.. ts:cv:: CONFIG proxy.config.output.logfile.rolling_max_count INT 0
+   :reloadable:
+
+   Specifies the maximum count of rolled output logs to keep. This value will be used by the
+   auto-deletion (if enabled) to trim the number of rolled log files every time the log is rolled.
+   A default value of 0 means auto-deletion will not try to limit the number of output logs.
+   See :doc:`../logging/rotation.en` for an use-case for this option.
+
+.. ts:cv:: CONFIG proxy.config.output.logfile.rolling_allow_empty INT 0
```

Comment:
I would set this to be on by default and not have a configuration option for it.

Reply:
Sounds reasonable to me, other solutions like i.e. [multilog](https://cr.yp.to/daemontools/multilog.html) have similar behavior by default.

But  from traffic server logging code it seems to me that someone has gone great lengths to make sure no empty log files are rolled, so having an option to enable/disable this behavior made more sense to me.